SignalServiceKit/src/Contacts/OWSContactDiscoveryOperation.swift

```diff
@@ -83,13 +85,15 @@ class LegacyContactDiscoveryBatchOperation: OWSOperation {
 
     // Called at most one time.
     override func didSucceed() {
-        // Compare against new CDS service
-        let modernCDSOperation = CDSOperation(recipientIdsToLookup: self.recipientIdsToLookup)
-        let cdsFeedbackOperation = CDSFeedbackOperation(legacyRegisteredRecipientIds: self.registeredRecipientIds)
-        cdsFeedbackOperation.addDependency(modernCDSOperation)
-
-        let operations = modernCDSOperation.dependencies + [modernCDSOperation, cdsFeedbackOperation]
-        CDSOperation.operationQueue.addOperations(operations, waitUntilFinished: false)
+        if isCDSEnabled {
```

Comment:
nit: for more semantic diffs, a guard might be better, but TIOLI obvs

Reply:
Done.